src/SourceConstraints.hs

```diff
@@ -228,15 +214,10 @@ sortedLocated name context@Context{..} ordering nodes =
         dynFlags
         (getLoc actualNode)
         neverQualify
-        (text $ message expectedNode)
-
-    message :: Located a -> String
-    message expected =
-        "Unsorted " <> name <> ", expected: " <> render context expected
+        (text $ "Unsorted " <> name <> ", expected: " <> render context expectedNode)
 
     isViolation :: ((Located a, b), (Located a, b)) -> Bool
-    isViolation ((_actualNode, item), (_expectedNode, expected)) =
-      item /= expected
+    isViolation ((_actualNode, item), (_expectedNode, expected)) = item /= expected
```

Comment:
@mbj This hunk is unrelated to the `Change`. What about formatting it this way right when the function is introduced in the previous commit?

Reply:
worse it violates 79chars, I could just kill this diff.